posthog/models/web_preaggregated/sql.py

```diff
@@ -40,10 +39,10 @@ def HOURLY_TABLE_TEMPLATE(table_name, columns, order_by, on_cluster=True, ttl=No
         team_id UInt64,
         host String,
         device_type String,
-        updated_at DateTime64(6, 'UTC') DEFAULT now(),
         {columns}
     ) ENGINE = {engine}
     ORDER BY {order_by}
+    PARTITION BY formatDateTime(period_bucket, '%Y%m%d%H')
```

Comment:
Those are used as an `intra-day` table to hold the current day's data in the same format as the historical pre-aggregated data. We query both using a view defined on `create_combined_view_sql`